### PR TITLE
Vue 3 Sample

### PR DESCRIPTION
**Pull request by JeffWinder**
Sample of UXP with Vue 3, based off the Vue 2 sample.

### CODE REVIEW

**Review by pklaschka — APPROVED** [review]
**Preface:** I'm not familiar enough with Vue to do a proper code review of the Vue-specific parts of this PR. I'll, therefore, review it from a "plain" JS POV and do a functional test. If it works, I'll trust @JeffWinder about the Vue-specific aspects of the PR.

## Code Review

- `uxp` is expected within scripts defined in the `package.json`, but the Dev Tools CLI isn't listed as development dependency => was already the case for previous samples. I'll open an issue, but don't see this as part of this PR's scope.

No further comments

## Functional test

Steps performed:
1. Downloaded the sample
2. As per the `README.md` (using the `npm`-based instructions): ran `npm install`
3. As per the `README.md` (using the `npm`-based instructions): ran `npm run build` => success
4. Added plugin in UDT using `dist/manifest.json`
5. Loaded the plugin

Plugin loaded successfully, everything appears to be working => functional test successfully completed 🙂 

## In conclusion
As far as I can tell, this PR looks good. Waiting for @amandahuarng to review and after that, I think we can merge this. Thanks for your contribution ☺️ !

**Comment by pklaschka** [comment]
The dependency comment is now tracked in #59 => officially out of scope for this PR

**Review by amandahuarng — COMMENTED** [review]
Looks good to me 👍
